middleware.ts

```diff
@@ -50,7 +50,7 @@ export const config = {
     "/staff/:path*",
     "/facility/:path*",
     "/profile/:path*",
-    "/api/staff/:path*",
+    // "/api/staff/:path*",
```

Comment:
There are 2 potential solutions to this first one is to change the location of this endpoint to be in the api/admin/staff directory directory and add an accessControl entry for the new endpoint with the rules that you want or another solution would be to update the accessControl rules for this api endpoint to include a GET request as well as include the ADMIN role